src/ImageSharp/Common/Constants.cs

```diff
@@ -0,0 +1,18 @@
+﻿// <copyright file="Constants.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Common
+{
+    /// <summary>
+    /// Common constants used throughout the project
+    /// </summary>
+    public static class Constants
```

Comment:
I would be inclined to make this `internal` can't see why we would want to expose as part of the public API.